src/main/target/YUPIF4/target.h

```diff
@@ -138,7 +138,7 @@
 #define VBAT_ADC_PIN            PC1
 #define RSSI_ADC_GPIO_PIN       PC0
 
-
+#define USE_DSHOT
```

Comment:
Probably could do with a `#define USE_ESC_TELEMETRY` here in order to enable ESC feedback.